medusa/classes.py

```diff
@@ -88,54 +89,128 @@ def __init__(self, episodes):
         # content
         self.content = None
 
+        # Result type like: nzb, nzbdata, torrent
         self.resultType = u''
 
+        # Store the parse result, as it might be useful for other information later on.
+        self.parsed_result = None
+
+        # Reference to the search_provider
+        self.provider = provider
+
+        # Raw result in a dictionary
+        self.item = None
+
+        # Search flag for specifying if we want to re-download the already downloaded quality.
+        self.download_current_quality = None
+
+        # Search flag for adding or not adding the search result to cache.
+        self.add_cache_entry = True
+
+        # Search flag for flagging if this is a same-day-special.
+        self.same_day_special = False
+
+        # Keep track if we really want to result.
+        self.result_wanted = False
+
+        # The actual parsed season.
+        self.actual_season = None
+
+        # The actual parsed episode.
+        self.actual_episodes = None
+
+        # Flag for multi-epp search result.
+        self.multi_epp = False
```

Comment:
I don't know. I think I had a use for it. But can't remember what it was. I'll remove it. As we can already see if it is a mult_epp result, evaluating self.actual_episodes.